lib/Driver/Driver.cpp

```diff
@@ -68,7 +68,7 @@ Driver::Driver(StringRef DriverExecutable,
   : Opts(createSwiftOptTable()), Diags(Diags),
     Name(Name), DriverExecutable(DriverExecutable),
     DefaultTargetTriple(llvm::sys::getDefaultTargetTriple()) {
-      
```

Comment:
Can you separate the whitespace changes into a separate commit?

Reply:
sure thing :)

Reply:
I split the commit into two changes now

Reply:
Hey what's ur plan with the fushciaOS? Will it replace Android? Or, be another platform?

Reply:
Sorry, I'm not qualified to speak to that and it's little off-topic for this PR.